src/UKHO.S100PermitService.Common/Models/AuthTokenDetails.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Models
+{
+    [ExcludeFromCodeCoverage]
+    public class AuthTokenDetails
```

Comment:
Just call this AuthToken, and move it to the Providers namespace

Reply:
Resolved